CMakeLists.txt

```diff
@@ -3,17 +3,22 @@
 cmake_minimum_required(VERSION 3.20)
 project(lbstd_version VERSION 0.0.1 LANGUAGES CXX)
 
-option(LBSTD_VERSION_NS_NAME "The enclosing namespace for all classes of std_version" "lbstd")
+set(LBSTD_VERSION_NS "lbstd" CACHE STRING "The enclosing namespace for all classes of std_version")
 option(LBSTD_VERSION_ENABLE_FMTLIB "Enable fmtlib formatters integration" ON)
 option(LBSTD_VERSION_ENABLE_STDFORMAT "Enable std::format integration")
 option(LBSTD_VERSION_ENABLE_TESTS "Enable tests" ON)
 
+if (LBSTD_VERSION_NS STREQUAL "")
```

Comment:
Maybe it makes sense to add a check that LBSTD_VERSION_NS is a valid string?

-DLBSTD_VERSION_NS=42 cmake will pass the namespace, but the build will fail because 42 is not a valid namespace.